src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -26,8 +26,19 @@
 
 #include "UfbxImporter.h"
 
+#define UFBX_NO_INDEX_GENERATION
+#define UFBX_NO_GEOMETRY_CACHE
+#define UFBX_NO_TESSELLATION
+#define UFBX_NO_SUBDIVISION
+#define UFBX_NO_SCENE_EVALUATION
+
+/* Do we want this? */
+#define UFBX_NO_ERROR_STACK
```

Comment:
The error stack gives an internal "call stack" of `ufbxi_check()` values in case a file fails to load. It usually gives some context of what went wrong. I originally thought that it'd be nifty that users with broken FBX files could include it in issues filed for ufbx, making it easier for me to diagnose/fix the issue. Using something like `ufbx_format_error()` prints the stack into a string buffer or you can inspect it manually via `ufbx_error.stack[]`. It adds some ~100kB to the final exectutable/library in strings though when I last checked.

Here's an example where an ASCII file fails due to a truncated string, it's not especially user friendly but can be handy if debugging (and it at least gives you *some* idea of what happened):
```
Trade::UfbxImporter::openData(): loading failed:  ufbx v0.1.1 error: Failed to load
  8032:ufbxi_ascii_next_token: c != '\0'
  8052:ufbxi_ascii_accept: ufbxi_ascii_next_token(uc, &ua->token)
  8212:ufbxi_ascii_parse_node: ufbxi_ascii_accept(uc, 'N')
  9058:ufbxi_parse_toplevel_child_imp: ufbxi_ascii_parse_node(uc, 0, state, p_end, buf, true)
  9158:ufbxi_parse_toplevel_child: ufbxi_parse_toplevel_child_imp(uc, state, &uc->tmp_parse, &end)
 12018:ufbxi_read_objects: ufbxi_parse_toplevel_child(uc, &node)
 12653:ufbxi_read_root: ufbxi_read_objects(uc)
 20281:ufbxi_load_imp: ufbxi_read_root(uc)
```

Reply:
Oh that sounds great, the verbose flag is exactly what I wanted! I might add some flags to `ufbx_format_error()` as I need to fix it up anyways, found out that it prints a dummy stack of `0: 0: 0:` if stacks are disabled which is not desirable.